modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GatlingCodegen.java

```diff
@@ -0,0 +1,344 @@
+package io.swagger.codegen.languages;
+
+import io.swagger.codegen.*;
+import io.swagger.models.*;
+import io.swagger.models.parameters.*;
+import io.swagger.models.properties.*;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.util.*;
+import java.io.File;
+
+public class GatlingCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
Shouldn't it extend `AbstractScalaCodegen` ?

Reply:
I started this project with the generated helper files.  I can change it to that.

Reply:
Thank you @cbornet